app/config/collections2.php

```diff
@@ -967,6 +967,131 @@
             ]
         ],
     ],
+    'buckets' => [
+        '$collection' => Database::COLLECTIONS,
+        '$id' => 'buckets',
+        '$permissions' => ['read' => ['*']],
+        'name' => 'Buckets',
+        'structure' => true,
+        'attributes' => [
+            [
+                '$id' => 'dateCreated',
+                'type' => Database::VAR_INTEGER,
+                'format' => '',
+                'signed' => false,
+                'size' => 0,
+                'required' => false,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'dateUpdated',
+                'type' => Database::VAR_INTEGER,
+                'size' => 0,
+                'format' => '',
+                'signed' => true,
+                'required' => false,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'enabled',
+                'type' => Database::VAR_BOOLEAN,
+                'signed' => true,
+                'size' => 0,
+                'format' => '',
+                'filters' => [],
+                'required' => true,
+                'array' => false,
+            ],
+            [
+                '$id' => 'name',
+                'type' => Database::VAR_STRING,
+                'signed' => true,
+                'size' => 128,
+                'format' => '',
+                'filters' => [],
+                'required' => true,
+                'array' => false,
+            ],
+            [
+                
+                'label' => 'Adapter',
```

Comment:
Label attribute is no longer in use, we should remove it entirely from this config file.